flow/scripts/synth.tcl

```diff
@@ -1,4 +1,4 @@
-set ::env(VERILOG_FILES) $::env(RESULTS_DIR)/1_synth.rtlil
+set ::env(RTLIL_FILE) $::env(RESULTS_DIR)/1_synth.rtlil
```

Comment:
Switch to proc w/arguments instead of passing information via RTLIL_FILE globals?